src/intro_gui.cpp

```diff
@@ -33,13 +39,207 @@
 
 #include "safeguards.h"
 
+
+/**
+ * A viewport command for the main menu background (intro game).
+ */
+struct IntroGameViewportCommand {
+	/** Horizontal alignment value. */
+	enum AlignmentH : byte {
+		LEFT,
+		CENTRE,
+		RIGHT,
+	};
+	/** Vertical alignment value. */
+	enum AlignmentV : byte {
+		TOP,
+		MIDDLE,
+		BOTTOM,
+	};
+
+	int command_index;   ///< Sequence number of the command (performance order).
```

Comment:
`performance`? Did you mean `preferred` or something?

Reply:
No, the order the commands are performed (executed) in.